static/css/legacy.less

```diff
@@ -1416,6 +1416,12 @@ tr {
       padding-left: 30px !important;
     }
   }
+  td.valign_mid{
```

Comment:
The `td` makes these overly specific - I think `.valign_mid` should do. If not please add an inline comment explaining why they are necessary.

Reply:
I think this is a step in the right direction but I suggest calling this class .column--aligned-middle to be consistent with the other BEM naming we've been using recently.

Reply:
Okay I will change these

Reply:
DONE